tests/integration-tests/conftest.py

```diff
@@ -574,6 +574,9 @@ def inject_additional_image_configs_settings(image_config, request):
     with open(image_config, encoding="utf-8") as conf_file:
         config_content = yaml.load(conf_file, Loader=yaml.SafeLoader)
 
+    if not dict_has_nested_key(config_content, ("Build", "Imds", "RequireImdsV2")):
```

Comment:
I am agnostic with regards to this, with @demartinofra we decided on the other way around, but I don't have strong feelings on this.

Reply:
However, note that not all the tests will run with IMDSv2, we do have tests than use IMDSv1 (the code above sets IMDSv2 by default only if it is not otherwise specified in the test configuration template).

Reply:
Ok, I got the point. Thanks

Reply:
The main reason why I was in favor of running all tests with IMDSv2 is that we guarantee that no pcluster functionality ever relies on IMDSv1 being enabled. Also note that when IMDSv1 is enabled also IMDSv2 works, they are not mutually exclusive. 
At the same time having a single test that checks the default setting should give us enough confidence that the IMDSv1 works as expected.